### PR TITLE
fix: fix dockerfile for automated builds

### LINKED ISSUE #414: Restore dockerization of autopush
Autopush has a Dockerfile that used to work, but hasn't been maintained. It should be updated with automatic trusted Docker repo builds.

### PR DESCRIPTION
**Pull request by bbangert**
Dockerfile and Makefile were both broken. This fixes the Dockerfile
and uses the latest PyPy 5.0 for building the container.

Closes #414 

@jrconlin r?

### CODE REVIEW

**Comment by bbangert** [comment]
Looks like the other Travis CI test is never going to run, for unknown reason it won't let us restart it.

File: Makefile [hunk 1]
```diff
@@ -2,8 +2,8 @@ SHELL := /bin/sh
 APPNAME = autopush
 DEPS =
 HERE = $(shell pwd)
-PTYPE=python
-BIN = $(HERE)/bin
+PTYPE=pypy
```
Comment (kitcambridge) [inline Makefile]: I wonder if we should eventually add a task that sets up a CPython virtualenv and symlinks `python` to `pypy` for local development.

File: Makefile [hunk 2]
```diff
@@ -18,7 +18,7 @@ BUILD_DIRS = bin build deps include lib lib64 lib_pypy lib-python\
 
 all:	build
 
-travis: $(HERE)/ddb
+travis: $(HERE)/ddb $(BIN)/tox
```
Comment (kitcambridge) [inline Makefile]: Hmm...don't we still need to install `$(BIN)/tox` at this point?

**Comment by kitcambridge** [comment]
Please ignore the previous comment about tox; I didn't see that we already have a task for `$(BIN)/tox`. r+

**Pull request merged**